frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -44,12 +46,17 @@ const SecretFieldIcon = (): JSX.Element => (
 export function PluginDrawer(): JSX.Element {
     const { user } = useValues(userLogic)
     const { editingPlugin, loading, editingSource, editingPluginInitialChanges } = useValues(pluginsLogic)
-    const { editPlugin, savePluginConfig, uninstallPlugin, setEditingSource, generateApiKeysIfNeeded } = useActions(
-        pluginsLogic
-    )
+    const {
+        editPlugin,
+        savePluginConfig,
+        uninstallPlugin,
+        setEditingSource,
+        generateApiKeysIfNeeded,
+        patchPlugin,
+    } = useActions(pluginsLogic)
     const [form] = Form.useForm()
 
-    const canDelete = user?.plugin_access.install
+    const canDelete = (user?.organization?.plugins_access_level ?? 0) >= PluginsAccessLevel.Install
```

Comment:
Will the delete button show for plugins of other organisations that are global?

Reply:
Not anymore